BeerMeUiKit/BeerModel.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  BeerModel.swift
+//  BeerMeUiKit
+//
+//  Created by Matilde Tognocchi on 21/2/24.
+//
+
+import Foundation
+
+struct BeerModel: Decodable {    
+    
+    let id: Int
+    let uid, brand, name, style: String
+    let hop, yeast, malts, ibu: String
+    let alcohol, blg: String
```

Comment:
```suggestion
    let uid: String 
    let brand: String
    let name: String
    let style: String
    let hop: String
    let yeast: String
    let malts: String
    let ibu: String
    let alcohol
    let blg: String
```

Se scrivi le property così, il file è più facile da leggere.
Inoltre, deserializzi tutte queste property? Se non le usi, meglio cancellare quelle che non utilizzi, [YAGNI](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiS3vvI8saEAxXs_rsIHdD9BbQQFnoECBIQAQ&url=https%3A%2F%2Fit.wikipedia.org%2Fwiki%2FYou_aren%2527t_gonna_need_it&usg=AOvVaw2-ZuH58DTmxrKOD4j78b3s&opi=89978449)!